assets/base.css

```
@@ -1200,21 +1196,41 @@ deferred-media {
   padding: var(--buttons-padding) calc(var(--buttons-padding) * 3);
   text-decoration: none;
   border: 1px solid transparent;
-  border-radius: var(--buttons-radius);
   background-color: rgba(var(--color-button), var(--alpha-button-background));
-  box-shadow: 0 0 0 var(--buttons-border-width) rgba(var(--color-button), var(--buttons-border-opacity));
   color: rgb(var(--color-button-text));
   min-width: 12rem;
   min-height: 4.5rem;
-  transition: box-shadow var(--duration-short) ease;
   -webkit-appearance: none;
   appearance: none;
+}
+
+.button,
+.shopify-challenge__button,
+.customer button,
+.shopify-payment-button__button--unbranded,
+.shopify-payment-button [role="button"] {
+  box-shadow: 0 0 0 var(--buttons-border-width) rgba(var(--color-button), var(--buttons-border-opacity));
+}
+
+.button,
+.shopify-challenge__button,
+.customer button,
+.shopify-payment-button__button--unbranded,
+.shopify-payment-button [role="button"],
+.cart__dynamic-checkout-buttons div[role='button'],
```

Do we need to specify `div` in the selector?
```suggestion
.cart__dynamic-checkout-buttons [role='button'],
```